### PR TITLE
Bmo/migrate

### PR DESCRIPTION
**Pull request by Duckling92**
Migrates code from our private repo to a new public Glaze repo. Specifically,
- Scanning functionality is migrated, and
- `Pulse` and `UnprocessedWaveform` are migrated.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+[project]
+name = "pyglaze"
+version = "0.1.0"
+description = "Pyglaze is a library used to operate the devices of Glaze Technologies"
+readme = "README.md"
+license = { file = "LICENSE" }
+authors = [
+    {name = "GLAZE Technologies ApS"},
+]
+requires-python = ">=3.9"
```
Comment (ehrhorn) [inline pyproject.toml:10]: Is it maybe a good idea to require less than 3.13, which will release at some point and will probably be un-tested?

File: requirements-dev.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+ruff>=0.1.7
```
Comment (ehrhorn) [inline requirements-dev.txt:1]: Are requirements files still necessary? Can we not use standard pip and groups?

**Review by ehrhorn — CHANGES_REQUESTED** [review]
Two very small comments, one of them probably not even related to this PR (and should be fixed later if possible)

File: requirements-dev.txt (hunk 2 above)
Comment (Duckling92) [reply requirements-dev.txt:1]: It is not possible with a pyproject.toml-based requirements file to only install an additional dependency group without also installing the main dependencies, hence this approach.

**Review by ehrhorn — APPROVED** [review]

**Pull request merged**